code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -41,14 +41,18 @@
 			user << "\red The Bluespace generator isn't working."
 			return
 		if(istype(W, /obj/item/weapon/storage/backpack/holding) && !W.crit_fail)
-			investigate_log("has become a singularity. Caused by [user.key]","singulo")
-			user << "\red The Bluespace interfaces of the two devices catastrophically malfunction!"
-			qdel(W)
-			var/obj/singularity/singulo = new /obj/singularity (get_turf(src))
-			singulo.energy = 300 //should make it a bit bigger~
-			message_admins("[key_name_admin(user)] detonated a bag of holding")
-			log_game("[key_name(user)] detonated a bag of holding")
-			qdel(src)
+			user.visible_message("<span class='warning'>[user] grins as he begins to put a Bag of Holding into a Bag of Holding!</span>", "<span class='warning'>You begin to put the Bag of Holding into the Bag of Holding!</span>")

```

Comment:
Something like:
var/response = alert(src, "Are you -sure- you want to put the bag of holding inside another bag of holding?\n(This is a good way to become VERY, VERY dead)","Are you sure you want to die?","Live","Die")

And then before the do_after, if they chose "live", just return from the function, which will leave things the way they are, without making everyone superded